lib/twit.rb

```diff
@@ -0,0 +1,13 @@
+class Twit
+  attr_reader :client
+
+  def initialize
+    @client = Twitter::REST::Client.new do |config|
+      config.consumer_key = ENV['TWITTER_CONSUMER_KEY']
+      config.consumer_secret = ENV['TWITTER_CONSUMER_SECRET']
+      config.access_token = ENV['TWITTER_ACCESS_TOKEN']
+      config.access_token_secret = ENV['TWITTER_ACCESS_TOKEN_SECRET']
+    end
+  end
```

Comment:
I really really like this pattern. Good place to hang functionality, good isolation of responsibility, and pretty easy to test. I'd like to see something similar for the other APIs you're using, and I'd like to see your interactions with `@client` isolated in this object as well. Grab me if you wanna discuss this more.